src/pages/login.js

```diff
@@ -135,7 +136,13 @@ const LoginForm = () => {
           isInvalid={errors.password}
         />
 
-        <RegisterButton type="submit">Sign In</RegisterButton>
+        {loading ? (
+          <CenterContainer>
+            <Loader type="TailSpin" color="#570EF1" height={26} width={26} />
+          </CenterContainer>
```

Comment:
I don't think CenterContainer was imported in the login.js file because I'm getting this error when pressing the "Sign in" button:
```
Unhandled Runtime Error
ReferenceError: CenterContainer is not defined
```